CppCoreGuidelines.md

```diff
@@ -13279,7 +13279,7 @@ It also avoids brittle or inefficient workarounds. Convention: That's the way th
         int sz;
     };
 
-    vector<double> v(10);
+    Vector<double> v(10);
```

Comment:
I think this was merged already anyway. In this case, I believe the example introduces a custom "Vector" class, and uses it (need to expand to see the whole example on Github).